doc/languages.rst

```diff
@@ -929,6 +929,14 @@ to view the docstring of the syntax checker.  Likewise, you may use
          The coding standard, either as name of a built-in standard, or as path
          to a standard specification.
 
+   .. syntax-checker:: psalm
+
+      Lint with `Psalm static analysis tool <https://psalm.dev/>`_.
+
+      .. defcustom:: flycheck-psalm-threads
```

Comment:
Maybe it doesn't make sense to specify the number of threads to analyze a single file.